grails-app/conf/application.yml

```diff
@@ -244,6 +244,7 @@ citation.link.template: "For more information: @link@"
 citation.rights.template: ""
 
 siteDefaultLanguage: "en"
+showDataResourceWebsiteBelowH3: true
```

Comment:
Should this be false? Default in code is false. Maybe check prod ansible code?

Reply:
Updated